src/ansys/fluent/core/filereader/casereader.py

```diff
@@ -170,3 +175,35 @@ def _get_processed_string(input_string: bytes) -> str:
     rp_vars_str = codecs.decode(input_string, errors="ignore")
     string_identifier = "(37 ("
     return string_identifier + rp_vars_str.split(string_identifier)[1]
+
+
+def _get_case_filepath(project_dir_path: str) -> str:
+    """Gets path of any file having extension either of .cas, .cas.gz or.
+
+    .cas.h5 and returns the same.
```

Comment:
```suggestion
    """Gets case file path within the provided project directory path.
```

(@prmukherj I think there's an extra unneeded empty line, and maybe we don't need to document it as it stands. )